test/util/graph_verifier.h

```diff
@@ -51,10 +51,24 @@ class GraphVerifier {
 
 class BadEdgeException : public exception {
   virtual const char* what() const throw() {
-    return "The edge is not in the cut of the sample!";
+    return "The edge is not in the cut of the sample! (standard)";
   }
 };
 
+
+// class BadEdgeException1 : public exception {
+//   virtual const char* what() const throw() {
+//     return "The edge is not in the cut of the sample! (1)";
+//   }
+// };
+
+// class BadEdgeException2 : public exception {
+//   virtual const char* what() const throw() {
+//     return "The edge is not in the cut of the sample! (2)";
+//   }
+// };
+
+
```

Comment:
no need for this since we have something similar in `graph_verifier.cpp`

Reply:
Ok